README.md

```diff
@@ -0,0 +1,71 @@
+# Flocx-ui
+
+Flocx plugin for horizon to show and create bids, offers, and contracts
+
+* Free software: Apache license
+* Release Notes: View the [changelog](CHANGELOG.md)
+* Contributing: Read [CONTRIBUTING.md](CONTRIBUTING.md)
+
+## Enabling in DevStack
+
+Add this repo as an external repository into your `local.conf` file:
+
```

Comment:
Entering all the local.conf configurations will be better. 
ADMIN_PASSWORD=<Password for admin privilege> 
DATABASE_PASSWORD=$ADMIN_PASSWORD
RABBIT_PASSWORD=$ADMIN_PASSWORD
SERVICE_PASSWORD=$ADMIN_PASSWORD